API-Samples/Tutorial/markdown/10-init_render_pass.md

```diff
@@ -90,6 +91,11 @@ larger than 2x2.
 GPU hardware that has the ability to use optimal image layouts
 also has controls for deciding between using linear and optimal
 layouts.
+Although some GPU hardware may be forced to use a particular
+layout for some operations.
+For example, a GPU may only be able to render to optimal
```

Comment:
I think using IMAGE_LAYOUT_GENERAL is ok everywhere, possibly with performance issues, but I don't think optimal is ever required